sg_jira/bridge.py

```diff
@@ -95,7 +95,11 @@ def __init__(
         shotgun.setup()
 
         self._jira_user = jira_user
-        self._jira = JiraSession(jira_site, basic_auth=(jira_user, jira_secret),)
+        if jira_user == "None":
+            options = dict(token_auth=jira_secret)
+        else:
+            options = dict(basic_auth=(jira_user, jira_secret))
```

Comment:
Why use `dict` instead of `{}`?

Then, I would go for something like:
```python
jira_options = {}
if jira_user == "None":
    jira_options["token_auth"] = jira_secret
else:
    jira_options["basic_auth"] = (jira_user, jira_secret)
```

Not a blocker of course. Just personal taste ;)

Reply:
Speaking of personal taste, since I advocate in favor of immutability I refactored this conditional using ternary conditional 😉